hoi/core/tests/test_entropies.py

```diff
@@ -0,0 +1,73 @@
+import pytest
+import numpy as np
+import jax
+from hoi.core.entropies import (
+    entropy_gcmi,
+    entropy_bin,
+    entropy_knn,
+    entropy_kernel,
+    copnorm_nd,
+    ctransform,
+)
+from hoi.utils import digitize
+
+x1 = np.random.rand(1, 50)
+x2 = np.random.rand(10, 50)
+# l1 = [[np.random.random() for _ in range(1)] for _ in range(50)]
+# l2 = [[np.random.random() for _ in range(10)] for _ in range(50)]
+j1 = jax.random.uniform(jax.random.PRNGKey(0), shape=(1, 50))
+j2 = jax.random.uniform(jax.random.PRNGKey(0), shape=(10, 50))
+
+
+# Smoke tests
+class TestEntropy(object):
```

Comment:
Few comments about testing entropy functions :
- There are still functions and not class methods like `test_entropy_kernel`, `test_entropy_knn` etc.
- Here, you only test output type. Output shape is also essential

Reply:
- Sorry, I didn't notice they weren't class methods. I made the functions a part of the class now.
- About the output, it is a float in all cases, so would only `assert hx.dtype == np.float32` not work?

Reply:
What about `assert hx.shape == ()`?